### PR TITLE
Document mask convention

### PR DESCRIPTION
**Pull request by colehaus**
It seems worth documenting the convention explicitly since, unless I'm missing something, it's the opposite of e.g. [PyTorch](https://pytorch.org/docs/stable/generated/torch.nn.MultiheadAttention.html)'s.

### CODE REVIEW

**Comment by patrick-kidger** [comment]
LGTM! Thank you. And you're right, this is the opposite convention of PyTorch's.
